test/integration/mirror.spec.ts

```diff
@@ -257,7 +257,7 @@ describe('mirror', () => {
 		expect(comment.body).toEqual(body);
 	});
 
-	test('should mirror message insert on support threads', async () => {
+	test.only('should mirror message insert on support threads', async () => {
```

Comment:
You'll want to remove this `.only` before merging

Reply:
Nice catch!